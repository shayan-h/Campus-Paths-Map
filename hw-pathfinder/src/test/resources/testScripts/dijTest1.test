# Find a path with Dijkstra's algorithm when
# the node is incoming to itself.

CreateGraph simpleGraph

AddNode simpleGraph A

AddEdge simpleGraph A A 15.0

FindPath simpleGraph A A