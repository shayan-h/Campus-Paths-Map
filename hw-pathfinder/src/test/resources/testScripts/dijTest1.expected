# Find a path with Dijkstra's algorithm when
# the node is incoming to itself.

created graph simpleGraph

added node A to simpleGraph

added edge 15.000 from A to A in simpleGraph

path from A to A:
total cost: 0.000